src/main/java/ch/cern/eam/wshub/core/services/administration/UserGroupMenuService.java

```diff
@@ -0,0 +1,32 @@
+package ch.cern.eam.wshub.core.services.administration;
+
+import ch.cern.eam.wshub.core.annotations.Operation;
+import ch.cern.eam.wshub.core.client.InforContext;
+import ch.cern.eam.wshub.core.interceptors.LogDataReferenceType;
+import ch.cern.eam.wshub.core.services.INFOR_OPERATION;
+import ch.cern.eam.wshub.core.tools.InforException;
+
+
+public interface UserGroupMenuService {
+    class MenuSpecification {
+        public String userGroup;
```

Comment:
Following conventions, instance variables should be private and have getters/setters

Reply:
Solved.